private.yaml

```diff
@@ -17,23 +17,39 @@ metadata:
   name: velum-private
   labels:
     name: velum-private
+  annotations:
+    pod.beta.kubernetes.io/init-containers: '[
+      {
+        "name": "mariadb-secrets",
+        "image": "sles12/velum:development",
+        "command": ["sh", "-c", "if [ ! -f /infra-secrets/mariadb-password ]; then head -n 1000 /dev/random | sha256sum | cut -d\" \" -f1 > /infra-secrets/mariadb-password; fi; exit 0;"],
+        "volumeMounts": [
+          {
+            "mountPath": "/infra-secrets",
+            "name": "infra-secrets"
+          }
+        ]
+      }
+    ]'
```

Comment:
As far as I could see here https://kubernetes.io/docs/concepts/workloads/pods/init-containers/ this syntax is 1.6 only... for 1.5 (that is what we have in CaaSP right now) only the annotations syntax will work.

When we are stable with 1.6 we should move to the new syntax  though... that is much more convenient...